app/views/users/edit.html.erb

```diff
@@ -0,0 +1,52 @@
+<%= form_for @user, url: {action: "update"} do |f| %>
+
+    <% if @user.errors.none? %>
+      <%@clas1 = "default"%>
+      <%@clas2 = "default"%>
+      <%@clas3 = "default"%>
+    <% end %>
+
+    <% if @user.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@user.errors.count, "error") %> prohibited this user from being saved:</h2>
+
+      <ul>
+      <% @user.errors.full_messages.each do |msg| %>
+        <% edit_form_field_errors(msg) %>
+        <li><%= msg %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
```

Comment:
Please remove this logic from the view and put it in a helper, as the previous reviewer instructed.

That's it!